### PR TITLE
first results

### PR DESCRIPTION
**Pull request by Jahenstein**
Jeszcze nie wszystko zrobione

### CODE REVIEW

**Comment by Jahenstein** [comment]
Alembic added

**Comment by Jahenstein** [comment]
pytest script works, everything is done

File: test/README.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+in terminal:
+pytest test.py
```
Comment (etude-ist) [inline test/README.txt]: Only pytest should work.

File: test/test.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from urllib.request import urlopen
+
+def test():
+    assert urlopen('http://localhost:5000/').getcode()==200
```
Comment (etude-ist) [inline test/test.py]: Reimplement that using pytest tools.

File: test/README.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+in terminal:
```
Comment (etude-ist) [inline test/README.txt]: Let's use markdown format for this kind of docs.

File: config.py [hunk 4]
```diff
@@ -9,8 +9,7 @@ class Config(object):
 
 class DevConfig(Config):
     DEBUG = True
-    DATABASE_URI = getenv('DEV_DATABASE_URI', '')
-
+    SQLALCHEMY_DATABASE_URI=getenv('postgres://Lukasz:pass:@localhost:5000','')
```
[the remark this answers is not in the document]
Comment (etude-ist) [reply config.py]: Let's decompose that. See other pul requests for reference. This was already suggested in other pull requests.

**Pull request closed without merging**